docs/section4/4-10.md

```diff
@@ -0,0 +1,43 @@
+**Appendix G Section:** Table G3.1-4 Schedules for the proposed building excluding exceptions #s 2 and 3  
+**Appendix G Section Reference:** None  
+**Data Lookup:** None  
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem  
+**Applicability Checks:**  
+
+1. Not applicable to HVAC systems in which there are health and safety mandated minimum ventilation requirements during unoccupied hours.
+2. Not applicable to HVAC systems primarily serving computer rooms. 
+ 
+**Function Call:** 
+
+1. get_zones_computer_rooms()
+2. get_zones_health_safety_vent_reqs()
+
+**Manual Check:** None 
+
+**Rule Logic**
+**Applicability Checks 1 and 2:** 
+- Get list of zones that have computer rooms associated with them: `zones_serving_comp_rooms_list_p = get_zones_computer_rooms(P_RMR)`
+- Get list of zones that are likely to have health and safety mandated minimum ventilation requirements during unoccupied hours: `zones_health_safety_unocc_list_p = get_zones_health_safety_vent_reqs(P_RMR)`
+- Combine lists and remove duplicates: `inapplicable_zones_list_p = list(set(zones_serving_comp_rooms_list_p + zones_health_safety_unocc_list_p))`
+- For each zone in P_RMR: `for zone_p in P_RMR..Zone:`
+    - Reset boolean variable that indicates if a zone serves computer room(s) or has health and safety reqs during unocc hours: `zone_serves_comp_room_health_safe_check = FALSE`
+    - Check if zone is found in the list of zones serving comp rooms and with health and safety reqs during unocc periods, if NOT found then the zone is applicable to this RDS: `if zone_p.id not in inapplicable_zones_list_p: zone_serves_comp_room_health_safe_check = TRUE`
+    - Check if the boolean variable that indicates if a zone serves computer room(s) or has health and safety reqs during unocc hours equals true and if so carry on with rule logic: `if zone_serves_comp_room_health_safe_check == TRUE:`
+        - For each terminal in zone_p: `for terminal_p in zone_p.terminals:`
+            - Get HVAC system serving terminal: `hvac_p = terminal_p.served_by_heating_ventilation_air_conditioning_systems`
+            - Add to list of heating_ventilation_air_conditioning_systems_p as the code loops through the terminal units: `heating_ventilation_air_conditioning_systems_list_p = heating_ventilation_air_conditioning_systems_list_p.append(heating_ventilation_air_conditioning_systems_p)`                    
+        - Add to master list of HVAC systems that are applicable to this check after looping through terminal units: `applicable_hvac_systems_list_p = applicable_hvac_systems_list_p.append(heating_ventilation_air_conditioning_systems_list_p)`               
+- Convert the list of applicable_hvac_systems_list_p to a set and the back to a list to eliminate duplicates after looping through all zones: `applicable_hvac_systems_list_p = list(set(applicable_hvac_systems_list_p))`                             
```

Comment:
Logic above would need to be changed to remove zones with computer rooms if recommendation above is followed, as the function would return the HVAC systems.